dagfactory/dagfactory.py

```diff
@@ -195,7 +195,7 @@ def load_yaml_dags(
         ))
     for config_file_path in candidate_dag_files:
         config_file_abs_path = str(config_file_path.absolute())
-        logging.info(f"Loading {config_file_abs_path}")
+        logging.info("Loading %s, config_file_abs_path")
```

Comment:
I think this should be:
```python
logging.info("Loading %s", config_file_abs_path)
```
As is the log line will be(approx)
`INFO:root:Loading %s, config_file_abs_path`

Reply:
You are right, good catch